### PR TITLE
Show vitals from the current visit at the top of the consult note

### PR DESCRIPTION
**Pull request by jmbabazi**

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/mexico/consult.xml [hunk 1]
```diff
@@ -191,6 +193,69 @@
         </p>
     </div>
 
+
+    <div class = "small-text section-container" style="color: #E94B3CFF;">
```
Comment (brandones) [inline omod/src/main/webapp/resources/htmlforms/mexico/consult.xml]: Let's leave the text in its default color. Red means "the patient is about to catch fire"

File: omod/src/main/webapp/resources/htmlforms/mexico/consult.xml [hunk 2]
```diff
@@ -191,6 +193,69 @@
         </p>
     </div>
 
+
```
Comment (brandones) [inline omod/src/main/webapp/resources/htmlforms/mexico/consult.xml]: Let's wrap this with a `<section>` at the top level. Right now it looks to me like it's bleeding off to the right (let me know that's not what you see on your screen).

File: omod/src/main/webapp/resources/htmlforms/mexico/consult.xml [hunk 3]
```diff
@@ -191,6 +193,69 @@
...
+    <div class = "small-text section-container" style="color: #E94B3CFF;">
+        <table>
+        <thead>
+            <tr>
+
+                <td style="font-weight:bold"> <uimessage code="Signos vitales"/> </td>
```
Comment (brandones) [inline omod/src/main/webapp/resources/htmlforms/mexico/consult.xml]: These should be actual `message.properties` codes.

File: omod/src/main/webapp/resources/htmlforms/mexico/consult.xml [hunk 4]
```diff
@@ -191,6 +193,69 @@
...
+                <td style="font-weight:bold"> <uimessage code="Fecha"/> </td>
+                <td style="font-weight:bold"> <uimessage code="Resultados"/> </td>
+            </tr>
+            </thead>
+        <tbody>
+            <tr>
+                <td>  <uimessage code="PA (S)"/>  </td>
```
Comment (brandones) [inline omod/src/main/webapp/resources/htmlforms/mexico/consult.xml]: This and "PA (D)" should also be real `message.properties` codes

**Comment by brandones** [comment]
A few things to fix, but overall I really like the approach you took!

**Comment by alaboso** [comment]
Is this in the header section or in individual htmlforms upper section?

I have always thought that having visit vitals in the patient header section or top level of a form would be helpful!

Nice one!

File: omod/src/main/webapp/resources/htmlforms/mexico/consult.xml [hunk 5]
```diff
@@ -191,6 +193,71 @@
...
+<section>
+    <div class = "small-text section-container">
+        <table>
+        <thead>
+            <tr>
+
+                <td style="font-weight:bold"> <uimessage code="mirebalais.mostRecentVitals.label"/> </td>
+                <td style="font-weight:bold"> <uimessage code="zl.date"/> </td>
+                <td style="font-weight:bold"> <uimessage code="mirebalais.vital.results"/> </td>
+            </tr>
```
Comment (brandones) [inline omod/src/main/webapp/resources/htmlforms/mexico/consult.xml]: Let's clean up a the indentation (just run auto-indent)

**Comment by brandones** [comment]
Looks great, thank you @jmbabazi ! Please merge when you're ready.

**Comment by jmbabazi** [comment]
Thank @brandones , I've merged it in

**Pull request merged**